docs/en/02-consensus.md

```diff
@@ -20,19 +20,11 @@ Ostracon's block generation mechanism based on Tendermint-BFT consists of the fo
 
 VRF is an algorithm for generating a hash value $t$ that can be used as a cryptographic pseudo-random number. The differences between VRF and typical hash functions or pseudo-random number generators are that only the owner of the private key can generate the hash value $t$, and anyone with the corresponding public key can verify the correctness of the hash value.
 
-A VRF hash generator $k$ generates a proof $\pi$ (VRF Proof) from the message $m$ using its private key $S_k$ as in Equation (1). Here, the hash value $t$ can be acquired from the proof $pi$ using Equation. (2). On the other hand, to verify that the hash value $t$ was generated by the owner of the private key $S_k$ based on the message $m$, the verifier applies the public key $P_k$ for $S_k$, $m$, and $\pi$ to Equation (3) to verify that both hash values are identical.
+A VRF hash generator $k$ generates a proof $\pi$ (VRF Proof) from the message $m$ using its private key $S_k$ as in Equation `1.`. Here, the hash value $t$ can be acquired from the proof $pi$ using Equation. `2.` On the other hand, to verify that the hash value $t$ was generated by the owner of the private key $S_k$ based on the message $m$, the verifier applies the public key $P_k$ for $S_k$, $m$, and $\pi$ to Equation `3.` to verify that both hash values are identical.
```

Comment:
I just noticed that in a few places in this document, `$pi$` $\pi$ is used where it should be `$pi$` $pi$. However, these have been corrected on the LBM documentation and will be fixed in the backport, so it doesn't seem necessary to fix them in this PR.

This document uses LaTeX notation because GitHub announced that it would support LaTeX notation in Markdown, but it was before they actually started supporting LaTeX, so it wasn't fully checked.